res/css/views/messages/_ReactionTooltipButton.scss

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 New Vector Ltd
+Copyright 2019 The Matrix.org Foundation C.I.C.
```

Comment:
Renaming a file still retains original copyright iirc

Reply:
Hmm, I created these files separately and I think Git just decided they might be a rename because a percentage of the content (the license header) is the same, so I think I'll leave the copyright as it is.